website/docs/docs/cloud/manage-access/set-up-sso-microsoft-entra-id.md

```diff
@@ -150,6 +150,12 @@ To complete setup, follow the steps below in the <Constant name="cloud" /> appli
 
 <Snippet path="login_url_note" />
 
+### Additional configuration options
+
+- **Include all groups:** Retrieve all groups to which a user belongs from your identity provider. If a user is a member of nested groups, it will also include the parent groups. When this option is disabled, only groups where the user has direct membership will be supplied.  This option is enabled by default.
+
+- **Maximum number of groups to retrieve:** Provides a configurable limit to the number of groups to retrieve for users.  By default, this is set to 250 groups, but this number can be increased if users' group memberships exceed that amount.
```

Comment:
What happens if the number of groups is set too low? How are the groups pulled into dbt decided? Is it a random sampling or based on some hierarchy?

Reply:
If the number of groups is set too low, then Entra will likely return an error that results in users seeing an error message displaying "Server error" in an error dialog box on their login page after the attempted login.